src/action/grpc.js

```diff
@@ -84,6 +84,17 @@ class GrpcAction {
     return this._sendIpc('lndRequest', 'lndResponse', method, body);
   }
 
+  /**
+   * Wrapper function to execute a call to the main process
+   * in order to retrieve the user's locale.
+   * @param  {string} method The lnd GRPC api to call
+   * @param  {Object} body   The payload passed to the api
+   * @return {Promise<Object>}
+   */
+  sendLocaleRequest() {
+    return this._sendIpc('get-locale', 'locale');
+  }
```

Comment:
options here:
1) make `_sendIpc` public so it can be called from `actions/setting`
2) replicate logic of `_sendIpc` in `actions/setting`, and pass `ipcRenderer` into that file
3) leave this code here, which seems out of place

I would prefer 1 or 3, just for aesthetics and simplicity